cmake/install.cmake

```diff
@@ -91,3 +91,14 @@ export(
   EXPORT singularity-eosTargets
   FILE ${CMAKE_CURRENT_BINARY_DIR}/singularity-eosTargets.cmake
   NAMESPACE singularity-eos::)
+
+# ----------------------------------------------------------------------------#
+# Data files
+# ----------------------------------------------------------------------------#
+
+# install data files needed for various eos models
+if(SINGULARITY_USE_HELMHOLTZ)
+  # install data files
+  install(DIRECTORY ${PROJECT_SOURCE_DIR}/data/helmholtz
+          DESTINATION ${CMAKE_INSTALL_DATADIR}/singularity-eos)
+endif()
```

Comment:
Raising a point here to check that this is correct - I need to look more closely but I'm worried that the install path will be `<install-prefix>/share/singularity-eos/helmholtz`, when the source expects `<install-prefix>/share/singularity-eos/data/helmholtz`

Reply:
Your're right, that was an oversight by me. Fixed.